content/en/docs/tasks/administer-cluster/cpu-management-policies.md

```diff
@@ -303,6 +304,21 @@ policy option is not compatible with `TopologyManager` `single-numa-node`
 policy and does not apply to hardware where the number of sockets is greater
 than number of NUMA nodes.
 
+
+If the `distribute-cpus-across-cores` policy option is specified, the static policy
+will attempt to allocate cpus (hyperthreads) across different physical cores.
```

Comment:
let's try to use the same terminology used by the rest of this document (please see the documentation of the other options above) and by the [resource management docs](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/)

For example, I'd write "...will attempt to allocate virtual cores (hardware threads) across different..."

Reply:
Got you. I updated the terms.